app/controllers/peer_reviews_controller.rb

```diff
@@ -112,8 +112,8 @@ def show_result
 
   def assign_groups
     @assignment = Assignment.find(params[:assignment_id])
-    selected_reviewer_group_ids = params[:selectedReviewerGroupIds] || []
-    selected_reviewee_group_ids = params[:selectedRevieweeGroupIds] || []
+    selected_reviewer_group_ids = Array(params[:selectedReviewerGroupIds]) || []
```

Comment:
I'm not sure why adding `Array` is needed here, as this PR isn't changing the front-end at all? If you're having trouble with the tests you've added, you can call `Array` in the tests themselves.

Reply:
That's exactly what it was, I was having issues with tests. There seemed to be some cases where the group ids weren't being treated as Arrays, but strangely I did not experience test failures after removing this change.
![image](https://github.com/user-attachments/assets/a41297f3-706f-41c7-a01e-e5dc85e55b72)